scripts/background.js

```diff
@@ -7,7 +7,7 @@ chrome.runtime.onInstalled.addListener(({ reason }) => {
   }
 });
 
-chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
+chrome.tabs.onUpdated.addListener(async (tabId, changeInfo, tab) => {
```

Comment:
I made this async when I added the tabs permission, but ultimately I don't think this is doing anything at all.